packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue

```diff
@@ -178,14 +177,18 @@
 													</div>
 												</div>
 											</section>
-											<Textarea
-												v-if="selectedItem === equation.id"
-												v-model="equation.asset.text"
-												autoResize
-												rows="1"
-												placeholder="Add an expression with LaTeX"
-												class="w-full overflow-y-scroll"
-											/>
+											<template v-if="selectedItem === equation.id">
+												<Textarea
+													v-model="equation.asset.text"
+													autoResize
+													rows="1"
+													placeholder="Add an expression with LaTeX"
```

Comment:
nitpick: "Add an expression with LaTeX" sounds a little weird, it is kind of like saying "add code snippet with javascript" ... I think "Add LaTeX expression" would be better.